src/shared/util/react-perf.js

```diff
@@ -0,0 +1,22 @@
+/*
+Copyright 2016 Mozilla
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+this file except in compliance with the License. You may obtain a copy of the
+License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed
+under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+*/
+
+if (process.env.NODE_ENV === 'development') {
+  module.exports = require('react-addons-perf'); // eslint-disable-line
```

Comment:
The button should also not be rendered in the UI if we aren't in dev since it won't do anything, right?